docs/source/external-tools/cdf/cdf_requirements.rst

```diff
@@ -20,7 +20,7 @@ ISTP Compliant Global Attributes are listed here: `https://spdf.gsfc.nasa.gov/is
 
 ``Data_type``
 =============
-This attribute is used by CDF file writing software to create a filename. It is a combination of the following filename components: ``mode``, ``data level``, and optional data product ``descriptor`` (e.g. ``L1A-norm>Level-1A-normal-rate``)
+This attribute is used by CDF file writing software to create a filename. It is a combination of the following filename components: ``mode``, ``data level``, and optional data product ``descriptor``.  It contains both the short form and the long form of the name, separated by a '>' (e.g. ``L1A-norm>Level-1A normal rate``)
```

Comment:
Is it supposed to be "Level-1A" or would it be "Level 1A" without the hyphen, or "Level1A" all together?